test/testmarlin/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ SET_TESTS_PROPERTIES(t_marlinminusx PROPERTIES PASS_REGULAR_EXPRESSION "Example
 SET_TESTS_PROPERTIES( t_marlinminusx PROPERTIES PASS_REGULAR_EXPRESSION "<parameter name=\"RandomSeed\" value=\"1234567890\" />" )
 
 #find the marlintest library and set the MARLIN_DLL to it
-GET_TARGET_PROPERTY( MARLIN_DLL marlintest LOCATION )
+set( MARLIN_DLL ${CMAKE_BINARY_DIR}/lib/libmarlintest.so )
```

Comment:
is the `lib` guaranteed to work here for all OSs? I assume it is, because it's the build directory.

Reply:
I think so, at least on Alma9 which is the problematic one with lib64 it appears in lib. Also I think the LCSoft packages hardcode lib somewhere.